src/models/analysis.js

```diff
@@ -3,17 +3,38 @@ import Config from '../config';
 import MusitActor from './actor';
 import MusitObject from './object';
 import { Observable } from 'rxjs';
+import moment from 'moment';
 
 class Analysis {}
 
-Analysis.fromJsonToForm = jsonObj =>
-  Object.keys(jsonObj).reduce(
-    (acc, attributeName) => [
-      ...acc,
-      { name: attributeName, defaultValue: jsonObj[attributeName] }
-    ],
-    []
+Analysis.fromJsonToForm = (
+  json: any
+): Array<{ name: string, defaultValue: string | boolean }> => {
+  // auxiliary method for converting to field
+  const toField = (name, defaultValue) => ({ name, defaultValue });
+  // Convert json to an object of form fields
+  const form = Object.keys(json).reduce(
+    (acc, key) => ({ ...acc, [key]: toField(key, json[key]) }),
+    {}
   );
+  // custom mapping of composite structures from json to form
+  form.caseNumber = toField(
+    'caseNumber',
+    form.caseNumbers ? form.caseNumbers.defaultValue[0] : null
+  );
+  const restriction = form.restriction && form.restriction.defaultValue;
+  form.restrictiona = toField('restrictions', !!restriction);
+  if (restriction) {
+    form.requester = toField('requester', restriction.requester);
```

Comment:
We should prefix the field when we're normalising them to prevent naming conflicts.